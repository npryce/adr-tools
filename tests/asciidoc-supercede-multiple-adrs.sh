echo "adoc" > .adr-file-type
adr new First Record
adr new Second Record
adr new -s 1 -s 2 Third Record
head -8 doc/adr/0001-first-record.adoc
head -8 doc/adr/0002-second-record.adoc
head -12 doc/adr/0003-third-record.adoc
