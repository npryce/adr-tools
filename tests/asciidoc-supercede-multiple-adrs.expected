echo "adoc" > .adr-file-type
adr new First Record
doc/adr/0001-first-record.adoc
adr new Second Record
doc/adr/0002-second-record.adoc
adr new -s 1 -s 2 Third Record
doc/adr/0003-third-record.adoc
head -8 doc/adr/0001-first-record.adoc
= 1. First Record

Date: 1992-01-12

== Status

Superceded by link:0003-third-record.adoc[3. Third Record]

head -8 doc/adr/0002-second-record.adoc
= 2. Second Record

Date: 1992-01-12

== Status

Superceded by link:0003-third-record.adoc[3. Third Record]

head -12 doc/adr/0003-third-record.adoc
= 3. Third Record

Date: 1992-01-12

== Status

Accepted

Supercedes link:0001-first-record.adoc[1. First Record]

Supercedes link:0002-second-record.adoc[2. Second Record]

